packages/ra-i18n-i18next/src/index.stories.tsx

```diff
@@ -0,0 +1,177 @@
+import * as React from 'react';
+import { Admin, EditGuesser, ListGuesser, Resource } from 'react-admin';
+import i18n from 'i18next';
+import resourcesToBackend from 'i18next-resources-to-backend';
+import englishMessages from 'ra-language-english';
+import fakeRestDataProvider from 'ra-data-fakerest';
+import { MemoryRouter } from 'react-router-dom';
+import { useI18nextProvider } from './index';
+import { convertRaTranslationsToI18next } from './convertRaMessagesToI18next';
+
+export default {
+    title: 'ra-i18n-18next',
+};
+
+export const Basic = () => {
+    const i18nProvider = useI18nextProvider({
+        options: {
+            resources: {
+                en: {
+                    translation: convertRaTranslationsToI18next(
+                        englishMessages
+                    ),
+                },
+            },
+        },
+    });
+
+    if (!i18nProvider) return null;
+
+    return (
+        <MemoryRouter>
+            <Admin dataProvider={dataProvider} i18nProvider={i18nProvider}>
+                <Resource
+                    name="posts"
+                    list={<ListGuesser enableLog={false} />}
+                    edit={<EditGuesser enableLog={false} />}
+                />
+                <Resource
+                    name="comments"
+                    list={<ListGuesser enableLog={false} />}
+                    edit={<EditGuesser enableLog={false} />}
+                />
+            </Admin>
+        </MemoryRouter>
+    );
+};
+
+export const WithLazyLoadedLanguages = () => {
+    const i18nextInstance = i18n.use(
+        resourcesToBackend(language => {
+            if (language === 'fr') {
+                return import(
+                    `ra-language-french`
+                ).then(({ default: messages }) =>
+                    convertRaTranslationsToI18next(messages)
+                );
+            }
+            return import(`ra-language-english`).then(({ default: messages }) =>
+                convertRaTranslationsToI18next(messages)
+            );
```

Comment:
It would be nice to have a little more of the UI translated, like resource names and column titles. It's a bit surprising for an user trying out the story to only have the Export button translated.

Reply:
Added